src/pykiso/pytest_plugin/collection.py

```diff
@@ -0,0 +1,331 @@
+##########################################################################
+# Copyright (c) 2010-2023 Robert Bosch GmbH
+# This program and the accompanying materials are made available under the
+# terms of the Eclipse Public License 2.0 which is available at
+# http://www.eclipse.org/legal/epl-2.0.
+#
+# SPDX-License-Identifier: EPL-2.0
+##########################################################################
+
+"""
+Test collection
+***************
+
+:module: collection
+
+:synopsis: allow pytest to collect test suites based on pykiso YAML
+    configuration files. Make defined auxiliaries available as fixtures.
+
```

Comment:
.. current_module:: seems to be missing from all your files

Reply:
Yep, removed it intentionally because it doesn't change anything and isn't required in that scope, see https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#directive-py-currentmodule